code/index.html

```diff
@@ -28,9 +28,12 @@
       </article>
     </section>
 
-    <h1 id="mystery"></h1>
+    <h1 id="theMystery"></h1>
 
-    <button>REVEAL THE CRIME</button>
+    <section class="btn">
+      <button id="revealBtn">REVEAL THE CRIME</button>
+      <button id="resetBtn">RESET</button>
+    </section>
 
     <script src="clue.js"></script>
   </body>
```

Comment:
All in all a very easy-to-read code!⭐️  Well done! Although this code does not really need comments, perhaps it is a good idea to start inserting comments in your html anyway - just to get into the habit of doing it.